ignite/metrics/regression/fractional_bias.py

```diff
@@ -64,7 +64,7 @@ class FractionalBias(_BaseRegression):
 
     @reinit__is_reduced
     def reset(self) -> None:
-        self._sum_of_errors = torch.tensor(0.0, dtype=torch.double, device=self._device)
+        self._sum_of_errors = torch.tensor(0.0, device=self._device)
```

Comment:
Keep double precision if it was already done here, just replace with `self._double_dtype`